archive/Thematic/2021/Februrary/progressed/bridgecrew2/template.html

```diff
@@ -753,8 +763,24 @@ <h1 class="null" style="text-align: center;display: block;margin: 0;padding: 0;c
 
 															<td valign="top" class="mcnTextContent" style="padding: 0px 18px 9px;line-height: 150%;mso-line-height-rule: exactly;-ms-text-size-adjust: 100%;-webkit-text-size-adjust: 100%;word-break: break-word;color: #111111;font-family: 'Source Sans Pro', 'Helvetica Neue', Helvetica, Arial, sans-serif;font-size: 18px;text-align: left;">
 
-																XXXX
-
+																<p dir="ltr" style="text-align: left;line-height: 150%;margin: 10px 0;padding: 0;mso-line-height-rule: exactly;-ms-text-size-adjust: 100%;-webkit-text-size-adjust: 100%;color: #111111;font-family: 'Source Sans Pro', 'Helvetica Neue', Helvetica, Arial, sans-serif;font-size: 18px;"><span style="font-size:17px"><span style="font-family:trebuchet ms,lucida grande,lucida sans unicode,lucida sans,tahoma,sans-serif"><strong id="docs-internal-guid-f6b1a81c-7fff-34dc-bbec-422bbeefc6f2"><strong style="font-weight: bolder;">Busting 5 More Myths about DevOps</strong></strong></span></span></p>
+																<br/>
+																<ol>
+																	<li><strong style="font-weight: bolder;">DevOps Doesn’t Work for Enterprises</strong></li>
+																</ol>
+																<div dir="ltr" style="text-align: justify;"><span style="font-size:16px"><span style="font-family:trebuchet ms,lucida grande,lucida sans unicode,lucida sans,tahoma,sans-serif">DevOps is applicable to everyone regardless of their industry or the company size. The implementation of this approach is fundamentally a change of process and the trick is that large companies don't set up cozy teams that can develop, qualify, and deploy code independently. Therefore, unlike <a href="https://hackernoon.com/how-to-cultivate-a-collaborative-devops-culture-ds1x334q" target="_blank" style="mso-line-height-rule: exactly;-ms-text-size-adjust: 100%;-webkit-text-size-adjust: 100%;color: #111111;font-weight: bold;text-decoration: underline;">small and medium-sized organizations</a>, enterprises have to conduct an orchestra of hundreds of employees by applying a more methodical approach for how they create, qualify, and release the code.</span></span></div>
+																<br/>
+																<ol start="2">
+																	<li><strong style="font-weight: bolder;">DevOps is a Cure-It-All Approach</strong></li>
```

Comment:
Same issue

Reply:
use this https://github.com/atherdon/newsletters/tree/master/archive/Thematic/snippets#h2